src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -9,6 +9,12 @@ redirect_from:
   - /integrations/github-enterprise/
   - /integrations/github/
 description: "Learn more about Sentry's GitHub integration and how it can track and resolve bugs faster by using data from your GitHub commits, and streamline your triaging process by creating a GitHub issue directly from Sentry."
+
+<Note>
+
+If you're not sure which GitHub integration you need to configure, please take a look at the URL. The https://github.com/org-name is usually for GitHub-hosted accounts. The self-hosted GitHub Enterprise accounts typically have URLs that look more like this: https://github.org-name.com (or some other customization of the domain). If that's the case, you should configure the non-Enterprise GitHub integration.
```

Comment:
```suggestion
If you're not sure which GitHub integration you need to configure, please take a look at the URL. URLs like `https://github.com/org-name` are usually for GitHub-hosted accounts and should use the regular GitHub (not GitHub Enterprise) integration. See [Install](#install) for more details.

Self-hosted GitHub Enterprise accounts typically have URLs that look more like `https://github.org-name.com` (or some other customization of the domain). In this case, you should configure the [GitHub Enterprise integration](#github-enterprise).
```

The last sentence made it confusing to what you were referring to for the non-enterprise integration, but I think it was the github.com/org-name url? I reworded this and made it two paragraphs for clarity. Feel free to revise if I got it mixed up.

I would also move this note down to after the first sentence of the page (line 20) so it comes up when users need to make a decision.

Reply:
hi @vivianyentran this is perfect, thank you! yes, it would also make sense to move this down